### DISCUSSION TITLE
I cannot figure out how to run tests inside vscode in my sveltekit monorepo

### DISCUSSION THREAD

**Discussion by christophsturm** [vitest-dev/vscode#205] [Q&A]
I have a monorepo with multiple sveltekit apps and I cannot get the vscode vitest plugin to show them
to reproduce the problem I have created an example monorepo with 2 sveltekit apps:
https://github.com/christophsturm/mono-test

the tests run fine when i run them from the sveltekit projects directories, but in the vscode test explorer I see nothing:
<img width="354" alt="image" src="https://github.com/vitest-dev/vscode/assets/1999/66c0d0e5-cb42-49f1-86f3-a2c8842e8873">

Please can someone tell me what I am doing wrong?

**Comment by Metehan-Altuntekin** [comment]
I had the same problem and this is how it went: 
I needed to define [Vitest workspaces](https://vitest.dev/guide/workspace.html) and that went okay. It worked fine with the initial sum test but I needed it to work with Svelte. I can run the test that uses Svelte fine on terminal but the extension was throwing an error that "svelte/stores" wasn't found. But it was working fine on terminal.

**Reply by christophsturm** [reply]
my demo repo has a vitest workspace defined and still does not work

**Reply by Metehan-Altuntekin** [reply]
you need `vitest.config` inside the workspace you want to have tests. Check this repo out: https://github.com/cozemble/breezbook/tree/vitest-configurations/apps/booking

**Comment by Metehan-Altuntekin** [comment]
Related: 
- https://github.com/vitest-dev/vscode/issues/47
- https://github.com/vitest-dev/vitest/issues/4630

**Comment by Metehan-Altuntekin** [comment]
I have found that I can just open VS Code at the workspace directory and it just works! The initial configurations from Svelte Kit setup is enough for my case. This might be annoying for some that is working in multiple workspaces at the same time but enough for my use case.

This the link to the workspace in the monorepo in case anyone wants to take a look:
https://github.com/cozemble/breezbook/tree/main/apps/booking

**Reply by christophsturm** [reply]
thats a great workaround but I still think it should also work when I open the root of the monorepo

**Reply by Metehan-Altuntekin** [reply]
Adding `vitest.config.ts` to the workspace is the closest I got but even if I'd copy paste the same thing from auto generated `vite.config.ts` it doesn't work properly
